src/components/staffProfile/staffProfile.stories.tsx

```diff
@@ -0,0 +1,46 @@
+import { ComponentMeta, ComponentStory } from '@storybook/react'
+import React from 'react'
+
+import { StaffProfile } from '@/components/staffProfile'
+import mediaImage from '@/components/media/media_example.json'
+import { mediaImageDataService } from '@/components/media/dataService'
+
+const image = mediaImageDataService(mediaImage)
```

Comment:
Looks like we have a leftover `image` const from the previous refactor.

Reply:
ah good catch! let me nuke that